BetterMD/elements/symbol.py

```
@@ -3,6 +3,7 @@
 from ..markdown import CustomMarkdown
 from ..html import CustomHTML
 from ..rst import CustomRst
+from ..parse import HTMLParser, MDParser, RSTParser, ELEMENT, TEXT, Collection
```

Ruff (0.8.2)

`..parse.RSTParser` imported but unused

Remove unused import: `..parse.RSTParser`

(F401)